### PR TITLE
Dome test create customer

### PR DESCRIPTION
**Pull request by domejiraphon**

### CODE REVIEW

File: features/customers.feature [hunk 1]
```diff
@@ -3,8 +3,27 @@ Feature: The customer store service back-end
...
+    And I set the "password" to "1111"
+    And I set the "first_name" to "jiraphon"
+    And I set the "last_name" to "yenphraphai"
+    And I set the "street_address" to "1234"
+    And I set the "city" to "New York"
+    And I set the "state" to "NY"
+    And I set the "zip" to "10090"
+    And I set the "country" to "USA"
+    And I press the "Create Customer" button
+    Then I should see the message "Success"
```
Comment (ryanjcohen) [inline features/customers.feature:29]: I'm wondering if we should add in tests for the error conditions ("username" is taken, missing one customer attribute, and missing just one address attribute)

Comment (domejiraphon) [reply features/customers.feature:29]: Sure

File: features/customers.feature [hunk 2]
```diff
@@ -3,8 +3,48 @@ Feature: The customer store service back-end
...
+    When I visit the "Home Page"
+    And I set the "username" to "dome"
+    And I set the "password" to "1111"
+    And I set the "first_name" to "jiraphon"
+    And I set the "last_name" to "yenphraphai"
+    And I set the "street_address" to "1234"
+    And I press the "Create Customer" button
+    Then I should see the message "Invalid Address: missing city"
+    
+Scenario: Create a customer with Invalid address
```
Comment (ryanjcohen) [inline features/customers.feature:35]: We should just change this scenario to "valid"

File: dot-env-example [hunk 3]
```diff
@@ -1,2 +1,6 @@
 # Copy this file to .env to expose these environment variables
 FLASK_APP=service:app
+PORT=8080
+WAIT_SECONDS=60
+CLOUDANT_USERNAME=admin
```
Comment (ryanjcohen) [inline dot-env-example:5]: we can remove these two cloudant variables, since we're not using cloudant

**Review by ryanjcohen — APPROVED** [review]

**Pull request merged**